CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## 9.0.7
+
+### Bug Fixes
```

Comment:
I think this title should be "Breaking Changes". 
```suggestion
### Breaking Changes
```

Reply:
I explained in the description: 

> I updated the CHANGELOG to make this fix a bug fix instead of breaking, because it doesn't make the hosting apps fails to compile.

If we want to keep it a breaking change, we would need to bump the version to a major number (10.0). I don't think this break the hosting apps, so I updated it.

Reply:
> I explained in the description:

Sorry, I missed this.

> I don't think this break the hosting apps, so I updated it.

This makes sense. 👍